tmt/cli.py

```diff
@@ -775,22 +875,36 @@ def plans_show(context: Context, **kwargs: Any) -> None:
 @click.pass_context
 @filter_options
 @fmf_source_options
+@lint_options
+@fix_options
```

Comment:
Why are you adding `fix_options` here? We don't have any automated fixes for plans. Or do we? The same question for stories.

Reply:
I believe we don't have any - I felt it would be better to include the options to present the consistent set of options across all `lint` subcommands. If there are no automated fixes, those options would turn into no-op, but I don't see that as a problem. In my books, not having to remember which of 4 `lint` commands does not accept `--fix` is a good benefit, and once we add the first fix, it'd work out of the box, as we couldn't forget to add `--fix` option.

Reply:
Ok, so it will be just prepared. Understand. Fine with me.